tests/python/release_checklist/check_graph_time_layout.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+import os
+import random
+from argparse import Namespace
+from uuid import uuid4
+
+import rerun as rr
+import rerun.blueprint as rrb
+
+README = """\
+# Graph view
+
+Please check the following:
+* Run the graph view in an endless loop and see if how it looks good (TM).
+* Try scrubbing the timeline to see how the graph layout changes over time.
```

Comment:
I agree, this test might be a bit tricky to automate, since it's really hard to codify what "twitching" looks like. We still might be able to use screenshot tests after navigating to exact points on the timeline.

Nevertheless, I've clarified the text for the check to make it clearer what people should look for.